src/firefly/js/ui/tap/TapUtil.js

```diff
@@ -268,6 +269,29 @@ async function doLoadTapColumns(serviceUrl, schemaName, tableName) {
     }
 }
 
+export const supportsObsCoreMetadataLoad = (serviceUrl) => {
+    return true || serviceUrl.toLowerCase().includes('lsst.cloud'); //for testing compatibility with other TAP services
```

Comment:
I have left this on in case you want to test CADC - the options will populate in the instrument and collection fields in 5.5 min from the time you selected the service.

Note: MAST ObsCore won't work because it blocks any query running longer than 2min